### PR TITLE
libunibreak: update to 4.2.

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/libunibreak/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'libunibreak'
 pkgname=libunibreak
-version=4.1
+version=4.2
 revision=1
 build_style=gnu-configure
 short_desc="An implementation of the Unicode line breaking algorithm"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="zlib-acknowledgement"
 homepage=https://fbreader.org/content/fbreader-beta-linux-desktop""
```
Comment (jnbr) [inline srcpkgs/libunibreak/template]: please fix the quotes

**Comment by Piraty** [comment]
@diogoleal ping

File: srcpkgs/libunibreak/template [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'libunibreak'
 pkgname=libunibreak
-version=4.1
+version=4.2
 revision=1
 build_style=gnu-configure
 short_desc="An implementation of the Unicode line breaking algorithm"
```
Comment (abenson) [inline srcpkgs/libunibreak/template:6]: `short_desc="Iimplementation...."

File: srcpkgs/libunibreak/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/libunibreak/template:9]: Shouldn't homepage be http://vimgadgets.sourceforge.net/libunibreak/ ?

File: srcpkgs/libunibreak/template [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'libunibreak'
 pkgname=libunibreak
-version=4.1
+version=4.2
```
Comment (abenson) [inline srcpkgs/libunibreak/template:3]: 4.3 is out

**Pull request closed without merging**